docs/kubernetes/config_examples/f5-k8s_multi-tenant-2b_custB.yaml

```diff
@@ -0,0 +1,87 @@
+# Deploy a BIG-IP Controller instance in the custB namespace
+# Manages the customerB partition on the BIG-IP system
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: custB_k8s-bigip-ctlr
+  namespace: bigip-controllers
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: custB_k8s-bigip-ctlr
+      labels:
+        app: k8s-bigip-ctlr
+    spec:
+      serviceAccountName: bigip-ctlr-serviceaccountB
+      containers:
+        - name: custB_k8s-bigip-ctlr
+          image: "f5networks/k8s-bigip-ctlr:1.3.0"
+          env:
+            - name: BIGIP_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: custB_bigip-login
+                  key: username
+            - name: BIGIP_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: custB_bigip-login
+                  key: password
+          command: ["/app/bin/k8s-bigip-ctlr"]
+          args: [
+            "--bigip-partition=customerB",
+            "--bigip-url=10.190.24.171",
+            "--bigip-username=$(BIGIP_USERNAME)",
+            "--bigip-password=$(BIGIP_PASSWORD)",
+            "--namespace=custB",
+            "--pool-member-type=cluster",
+            ...
```

Comment:
We should only provide example manifests that work 100% if a customer is to download them and then try to deploy it. Remove line 39 ellipsis as it's not needed.